spec/spec.md

```diff
@@ -0,0 +1,210 @@
+
+There are two ways to associate Copyright and Licensing Information with a
+file.
+
+### Comment headers
+
+To implement this method, each plain text file that can contain comments MUST
+contain comments at the top of the file (comment header) that declare that
+file's Copyright and Licensing Information.
+
+If a file is not a plain text file or does not permit the inclusion of
+comments, the comment header that declares the file's copyright and licensing
+information SHOULD be in an adjacent file of the same name with the
+additional extension `.license` (example: `cat.jpg.license` if the original
+file is `cat.jpg`).
+
+The comment header MUST contain one or more `SPDX-Copyright` tags, and one or
+more `SPDX-License-Information` tags. A tag is followed by a colon, followed by
+a text value, and terminated by a newline.
+
+The `SPDX-Copyright` tag MUST be followed by a copyright notice as described
+in <!-- FIXME: Refer to an internal section -->.
+
+Instead of the `SPDX-Copyright` tag, the symbol `©`, or the word `Copyright` MAY
+be used, in which case a colon is not needed..
+
+The `SPDX-License-Identifier` tag MUST be followed by a valid SPDX License
+Expression describing the licensing of the file (example:
+`SPDX-License-Identifier: GPL-3.0-or-later OR Apache-2.0`). If separate sections
+of the file are licensed differently, a different `SPDX-License-Identifier` tag
+MUST be included for each section.
+
+An example of a comment header:
+
+```
+# SPDX-Copyright: 2016, 2018-2019 Jane Doe <jane@example.com>
+# SPDX-Copyright: 2019 Example Company
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+```
+
+### DEP5
+
+Alternatively, Copyright and Licensing Information MAY be associated with a
+file through a DEP5 file. The intended use case of this method is large
+directories where including a comment header in each file (or in `.license`
+companion files) is impossible or undesirable.
+
+The DEP5 file MUST be named `dep5` and stored in the `.reuse/` directory in the
+root of the Project (i.e. `.reuse/dep5`).
+
+<!-- TODO: How to do multiple licenses without AND and OR. -->
+
+The `License` tag MUST be followed by a valid SPDX License Expression describing
+the licensing of the associated files.
+
+The `Copyright` tag MUST be followed by a copyright notice as described in <!--
+FIXME: Refer to an internal section -->.
+
+An example of a DEP5 file:
+
+```
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Project
+Upstream-Contact: Jane Doe <jane@example.com>
+Source: https://example.com/jane/project
+
+Files: po/*
+Copyright: 2019 Translation Company
+License: GPL-3.0-or-later
+```
+
+## Format of copyright notices
+
+A copyright notice MUST be prefixed by a tag, symbol or word denoting a
+copyright notice as described in this specification.
+
+The copyright notice MUST contain the year(s) of publication, the name of the
+copyright holder, and the contact address of the copyright holder. The order of
```

Comment:
```suggestion
copyright holder(s), and the contact address of the copyright holder(s); whereby the name of the copyright holder(s) MUST be included. The order of
```

Reply:
Multiple copyright holders in a single copyright notice?

Reply:
Seen that happen before.

But more sane, you have unincorporated groups of copyright holders such as:

```
© 2001 Contributors to Project X (see file `AUTHORS`)
```

```
Copyright 1998-2019 Project Y
```

I’m fine with omitting the “(s)” in “holder(s)” though, if anyone feels strongly about that.